tests/deploy_test.go

```diff
@@ -121,6 +123,30 @@ func Test_Deploy_WithAnnotations(t *testing.T) {
 	}
 }
 
+func Test_DeploymentNamespaces(t *testing.T) {
+	defaultNamespace := "openfaas-fn"
+	expectedNamespaces := config.Namespaces
+	expectedNamespaces = append(expectedNamespaces, defaultNamespace)
+	actualNamespaces, err := config.Client.ListNamespaces(context.Background())
+
+	if err != nil {
+		t.Fatalf("Unablen to List Openfaas Namespaces %s", err)
+	}
+
+	if len(expectedNamespaces) != len(actualNamespaces) {
+		t.Fatalf("Expected %v but got %v", expectedNamespaces, actualNamespaces)
```

Comment:
a more specific message will be easier to read and debug if the test fails, we can also make the language more consistent with the other tests
```suggestion
		t.Fatalf("want %d namespace(s),  got %d namespace(s)", expectedNamespaces, actualNamespaces)
```